client/dashboard/app/queries/site-themes.ts

```diff
@@ -0,0 +1,14 @@
+import { fetchSiteThemesActive } from '../../data/site-themes';
+import type { Theme } from '../../data/site-themes';
+
+export const siteThemesActiveQuery = ( siteId: number ) => ( {
+	queryKey: [ 'site', siteId, 'themes', 'active' ],
+	queryFn: () => fetchSiteThemesActive( siteId ),
+} );
+
+export const isFSEActiveQuery = ( siteId: number ) => ( {
```

Comment:
I think we're not using "FSE" term anymore 😄 maybe just simply `isSiteUsingBlockThemeQuery()`? 🤔